common/model-views.cpp

```diff
@@ -2346,10 +2357,22 @@ namespace rs2
                 std::string ref_str = "N/A";
                 try
                 {
-                    auto pixel_ref = _reflectivity->get_reflectivity( noise_est, max_usable_range, ir_val );
-                    _stabilized_reflectivity.add( pixel_ref );
-                    auto stabilized_pixel_ref = _stabilized_reflectivity.get( 0.75f );
-                    ref_str = to_string() << std::dec << round( stabilized_pixel_ref * 100 ) << "%";
+                    auto samples_ratio = _reflectivity->get_samples_ratio();
```

Comment:
`// Don't show a reflectivity value until the history is full`

I don't like using a "ratio" to determine this logic... I'd prefer `if( _reflectivity->is_history_full() )`

Reply:
I added a new function called is_history_full(), however I kept the ratio function cause I need it to draw the progress